### ISSUE TITLE
Properly handle keywords for style map values

### ISSUE THREAD

**Issue by metasoarous** [weavejester/hiccup#159]
It looks from #80 (and some repl tinkering) like keywords generally get cast correctly. However, I've found that for style attributes, the conversion does not work. For example,

```
=> (hiccup/html [:div {:style {:display :none}} [:p :Hello]])
"<div style=\"display::none;\"><p>Hello</p></div>"
```

Note that Reagent does handle this properly. I think it would be nice to maintain parity, as it can lead to gotchas when you're rendering the code in both contents (as I do with [Oz](https://github.com/metasoarous/oz)). It also seems to me that this behavior would make hiccup more consistent with itself, as anywhere else I tested (including as top-level attr maps values), it processed correctly.

Thanks for your work!

**Comment by weavejester** [comment]
This sounds a reasonable change.

**Comment by davidperrenoud** [comment]
Related: https://github.com/weavejester/hiccup/pull/140

**Comment by weavejester** [comment]
Closed by #194.
